tools/gulp/tasks/release.ts

```diff
@@ -32,6 +27,10 @@ const themingEntryPointPath = join(COMPONENTS_DIR, 'core', 'theming', '_all-them
 // Output path for the scss theming bundle.
 const themingBundlePath = join(releasePath, '_theming.scss');
 
+// Glob that matches all files that might be imported multiple times.
+// Necessary for deduping inside of scss-bundle.
+const themingBundleDedupeGlob = join(COMPONENTS_DIR, '**/*.scss');
```

Comment:
I named it `themingBundleDedupeGlob` because I wanted to make it clear that this will be the array that is responsible for deduping.